Remove tab characters from Fortran files

underwoo opened the pull request:
Remove tab characters from Fortran files.

Fixes #185

edhartnett commented:
I suggest you add a test for this in your CI system. (Are you just using travis, or have you also started using Jenkins locally? This is the kind of thing that Jenkins is great for.)

On other projects with lots of people, we automatically trigger standard formatting of all code, including control of indentation, tabs, and unneeded white-space. I can help you set this up if you like.

underwoo commented:
>I suggest you add a test for this in your CI system. (Are you just using travis, or have you also started using Jenkins locally? This is the kind of thing that Jenkins is great for.) ...

@edhartnett for now we only have the TravisCI running tests.  We are looking at using the GitLab CI to launch tests on one of the compute cloud providers.  However, that will require some time.

We are working on a lightweight code standard.  When we have something we are ready to release, we will setup tests to check for any of our standard violations.

We have used Jenkins in the past (and I think some of the GFDL developers still do for their personal tests), but for many reasons —- including the difficulty of connecting it to our production system through the bastions —- we stopped using it.

underwoo commented:
@thomas-robinson I have removed the tab character from the test Fortran files.  I also went through all files, and removed any tabs that aren't required. 

I also removed all end of line whitespace (except where required).  Please review the changes again.
